### DISCUSSION TITLE
traffic jam API

### DISCUSSION THREAD

**Discussion by Mannshoch** [osmandapp/OsmAnd#20298] [Ideas]
I accidentaly sumbled upon a REST API from viasuisse
https://demo.trafficintelligence.ch/

May there is a way to include that into Osmand?

**Comment by ZeljkoBG** [comment]
If I read correctly, it is traffic for 3-4 countries?

**Reply by Mannshoch** [reply]
Not sure. An Implementation of this API https://trafficmaptcs.trafficintelligence.ch shows only Switzerland. 
If you assume it from the Language you could request. Switzerland has [four official national Languages](https://en.wikipedia.org/wiki/Switzerland#Languages) so many website run their website in three of this languages (de, fr, it). As Far ViaSuisse is a Daughter company from [Swiss Broadcasting Corporation (SRG SSR)](https://en.wikipedia.org/wiki/Swiss_Broadcasting_Corporation) I assume they are limited to Switzerland.

**Comment by phenosys** [comment]
I also would be interested in seeing Swiss traffic information, especially now that the TMC service has been stopped on Jan 1. 2025. But if I understand the website correctly:
https://viasuisse.ch/it-services-datenlieferung/
there is a yearly fee of 12'000 CHF per year for access to the API, and that doesn't include sharing the data...
